simd/unit_tests/TestSIMD.cpp

```diff
@@ -177,8 +176,29 @@ void host_check_binary_op_one_loader(BinaryOp binary_op, std::size_t n,
   }
 }
 
+template <class Abi, class Loader, class UnaryOp, class T>
+void host_check_math_op_one_loader(UnaryOp unary_op, std::size_t n,
+                                   T const* args) {
```

Comment:
Any particular reason why you preferred overloading rather than named functions unary/binary?

Reply:
No particular reason. Initially the same function was used to test both unary and binary ops, but for now they are separated. If renaming these functions according to their operations sounds better, I'll change them.

Reply:
That's just my preference.  You don't have to change.